themes/default_symbols.fish

```diff
@@ -33,6 +33,9 @@ set FLSYM_ROOT_USER         "\u2192"
 # Symbol for VFISH segment
 set FLSYM_VFISH             "\u2635"
 
+# Symbol for CONDA segment
+set FLSYM_CONDA             "\U0001F40D"
```

Comment:
That's perfect ! Thank you, I am going to merge this to dev and release a v3.1.0 quickly that will include that and a minor change to the `test/run.sh` script (to fix an issue on fish 2.6.0).